ui-tests/src/test/java/org/openmrs/reference/page/ManageProviderSchedulesPage.java

```diff
@@ -31,6 +33,11 @@ public void selectLocation(String location) {
         selectFrom(LOCATION, location);
         clickOn(LOCATION);
     }
+    public void clickOnNextWeekday() {
```

Comment:
you probably mean workDay that's incase NEXT_WORK_DAY is right.

Reply:
Yes, this will always be a workday and probably a Wednesday. it will never be a Sunday or a Saturday